log/src/main/java/no/nav/familie/log/filter/LogFilter.kt

```diff
@@ -35,16 +37,21 @@ class LogFilter(
         filterChain: FilterChain,
     ) {
         val userId = resolveUserId(httpServletRequest)
-        if (userId == null || userId.isEmpty()) {
+        val skalGenerereBrukerIdCookie = systemtype == NavSystemtype.NAV_SAKSBEHANDLINGSSYSTEM && userId.isNullOrEmpty()
+        if (skalGenerereBrukerIdCookie) {
             // user-id tracking only works if the client is stateful and supports cookies.
             // if no user-id is found, generate one for any following requests but do not use it on the
             // current request to avoid generating large numbers of useless user-ids.
             generateUserIdCookie(httpServletResponse)
         }
         val consumerId = httpServletRequest.getHeader(NavHttpHeaders.NAV_CONSUMER_ID.asString())
         val callId = resolveCallId(httpServletRequest)
+
+        if(systemtype == NavSystemtype.NAV_SAKSBEHANDLINGSSYSTEM || systemtype == NavSystemtype.NAV_INTEGRASJON) {
```

Comment:
Kan skrives som
`if (systemtype in listOf(NavSystemtype.NAV_SAKSBEHANDLINGSSYSTEM, NavSystemtype.NAV_INTEGRASJON)) `

Reply:
Bra forslag, jeg fikser! 🙌